packages/css/src/components/heading/heading.scss

```diff
@@ -6,7 +6,7 @@
 @import "../../common/hyphenation";
 @import "../../common/text-rendering";
 
-@mixin reset {
+@mixin reset-heading {
```

Comment:
I understand this is a bit difficult. Options are `reset-hx`, `reset-hn`, and `reset-h`. Let’s go with the latter.